torchaudio/datasets/speechcommands.py

```diff
@@ -79,9 +92,22 @@ def __init__(self,
                     download_url(url, root, hash_value=checksum, hash_type="md5")
                 extract_archive(archive, self._path)
 
-        walker = walk_files(self._path, suffix=".wav", prefix=True)
-        walker = filter(lambda w: HASH_DIVIDER in w and EXCEPT_FOLDER not in w, walker)
-        self._walker = list(walker)
+        def load_list(filename):
+            filepath = os.path.join(self._path, filename)
+            with open(filepath) as fileobj:
+                return [os.path.join(self._path, line.strip()) for line in fileobj]
```

Comment:
Please prefer a plain function defined on module level especially when there is no local variable needs to be captured.

Also, since you are defining a helper function for your own sake, you can adopt a signature like

```
def _load_lists(*filename)
```

and it will make `excludes = load_lists(VALIDATION_LIST) + load_list(TESTING_LIST)` into as simple as `excludes = _load_lists("validation_list.txt", "testing_list.txt")`. (continue to the next comment)